packages/block-library/src/image/edit.native.js

```diff
@@ -432,197 +421,72 @@ export class ImageEdit extends React.Component {
-												),
-												{
-													height: imageContainerHeight,
-												},
-											] }
-										>
-											<View
-												style={
-													styles.imageUploadingIconContainer
-												}
-											>
-												{ this.getIcon(
-													ICON_TYPE.UPLOAD
-												) }
-											</View>
-										</View>
-									) }
-									<ImageBackground
-										accessible={ true }
-										disabled={ ! isSelected }
-										accessibilityLabel={ alt }
-										accessibilityHint={ __(
-											'Double tap and hold to edit'
-										) }
-										accessibilityRole={ 'imagebutton' }
-										style={ [
-											imageBorderOnSelectedStyle,
-											{
-												width: finalWidth,
-												height: finalHeight,
-												opacity,
-											},
-										] }
-										resizeMethod="scale"
-										source={ { uri: url } }
-										key={ url }
-									>
-										{ isUploadFailed && (
-											<View
-												style={ [
-													styles.imageContainer,
-													{
-														flex: 1,
-														backgroundColor:
-															'rgba(0, 0, 0, 0.5)',
-													},
-												] }
-											>
-												{ iconRetryContainer }
-												<Text
-													style={
-														styles.uploadFailedText
-													}
-												>
-													{ retryMessage }
-												</Text>
-											</View>
-										) }
-										{ isSelected &&
-											! isUploadInProgress &&
-											! isUploadFailed &&
-											finalWidth &&
-											finalHeight && (
-												<MediaEdit
-													allowedTypes={ [
-														MEDIA_TYPE_IMAGE,
-													] }
-													onSelect={
-														this
-															.onSelectMediaUploadOption
-													}
-													source={ { uri: url } }
-													openReplaceMediaOptions={
-														openMediaOptions
-													}
-													render={
-														editImageComponent
-													}
-												/>
-											) }
-									</ImageBackground>
-								</View>
-							);
-						} }
-					/>
-					<BlockCaption
-						clientId={ this.props.clientId }
-						isSelected={ this.state.isCaptionSelected }
-						accessible={ true }
-						accessibilityLabelCreator={ ( caption ) =>
-							isEmpty( caption )
-								? /* translators: accessibility text. Empty image caption. */
-								  'Image caption. Empty'
-								: sprintf(
-										/* translators: accessibility text. %s: image caption. */
-										__( 'Image caption. %s' ),
-										caption
-								  )
-						}
-						onFocus={ this.onFocusCaption }
-						onBlur={ this.props.onBlur } // always assign onBlur as props
-						insertBlocksAfter={ this.props.insertBlocksAfter }
-					/>
-				</View>
-			</TouchableWithoutFeedback>
+				</TouchableWithoutFeedback>
+				<BlockCaption
+					clientId={ this.props.clientId }
+					isSelected={ this.state.isCaptionSelected }
+					accessible={ true }
```

Comment:
nit: we don't need to pass true :) it's enough to just pass `accessible`

Reply:
Updated =)